templates/leaderboard/includes/list.html

```diff
@@ -1,18 +1,14 @@
+{% load staticfiles %}
 {% load humanize %}
 {% load perdiem %}
 
-{% if leaders %}
     <div class="leaderboard">
```

Comment:
Now that the giant `if leaders` block is removed, unindent this entire `<div>` block.